Tests/SonarQube.TeamBuild.PreProcessor.Tests/SonarLintAnalyzerProviderTests.cs

```diff
@@ -17,7 +17,8 @@ namespace SonarQube.TeamBuild.PreProcessor.Tests
     [TestClass]
     public class SonarLintAnalyzerProviderTests
     {
-        private const string DummyZippedBinaryFileName = "dummy.dll";
+        private const string DummyZippedBinaryFileName1 = "dummy1.dll";
+        private const string DummyZippedBinaryFileName2 = "dummy2.dll";
```

Comment:
The SonarLintAnalyzerProvider product code didn't need to change: it downloads a zip and extracts whatever the contents are. I've just updated the test to check that multiple files can be unpacked.